domain/schema/model/sql/0027-agent-version.sql

```diff
@@ -24,3 +24,11 @@ CREATE TABLE agent_version (
 -- A unique constraint over a constant index
 -- ensures only 1 row can exist.
 CREATE UNIQUE INDEX idx_singleton_agent_version ON agent_version ((1));
+
+CREATE TABLE latest_agent_version (
+    version TEXT NOT NULL
```

Comment:
I would just put this on the agent version table, it makes sense to live there in my opinion.